ospf/ietf-dynamic-flooding-ospf@2019-09-01.yang

```diff
@@ -1,476 +1,488 @@
-    }
-  }
-
-  /*
-   * Dynamic flooding state augmentation to OSPF3 module
-   */
-
-  augment "/rt:routing/rt:control-plane-protocols/"
-        + "rt:control-plane-protocol/ospf:ospf/ospf:areas/"
-        + "ospf:area/ospf:database/ospf:area-scope-lsa-type/"
-        + "ospf:area-scope-lsas/ospf:area-scope-lsa/"
-        + "ospf:version/ospf:ospfv3/ospf:ospfv3/ospf:body/"
-        + "ospf:router-information" {
-    if-feature "dynamic-flooding-ospf";
-    description
-      "dynamic flooding TLVs augmentation";
-    container area-leader-tlv {
-      description
-        "Area-leader-tlv";
-      uses ospf-area-leader-tlv;
-    }
-    container dynamic-flooding-tlv {
-      description
-        "dynamic-flooding-tlv";
-      uses dynamic-flooding-tlv;
-    }
-  }
-
-  augment "/rt:routing/rt:control-plane-protocols/"
-        + "rt:control-plane-protocol/ospf:ospf/ospf:areas/"
-        + "ospf:area/ospf:database/ospf:area-scope-lsa-type/"
-        + "ospf:area-scope-lsas/ospf:area-scope-lsa/"
-        + "ospf:version/ospf:ospfv3/ospf:ospfv3/ospf:body" {
-    if-feature "dynamic-flooding-ospf";
-    description
-      "ospfv3-dynamic-flooding-opaque-lsa
-       augmentation";
-    container ospfv3-dynamic-flooding-opaque-lsa {
-      description
-        "ospfv3-dynamic-flooding-opaque-lsa
-         definition";
-      //			uses unknown-tlvs;
-      container ospfv3-area-router-id-tlv {
-        description
-          "ospfv3-area-router-id-tlv";
-        uses ospfv3-area-router-id-tlv;
-      }
-      container flooding-path-tlv {
-        description
-          "ospf flooding-path-tlv";
-        uses flooding-path-tlv;
-      }
-    }
-  }
-}
+module ietf-dynamic-flooding-ospf {
+  yang-version 1.1;
+  namespace "urn:ietf:params:xml:ns:yang:ietf-dynamic-flooding-ospf";
+  prefix ospf-df;
+
+  import ietf-ospf {
+    prefix ospf;
+    reference
+      "RFC AAAA: YANG Data Model for OSPF Protocol";
+  }
+  import ietf-yang-types {
+    prefix yang;
+    reference
+      "RFC 6991: Common YANG Data Types";
+  }
+  import ietf-routing {
+    prefix rt;
+    reference
+      "RFC 8349 - A YANG Data Model for Routing
+       Management (NMDA Version)";
+  }
+
+  organization
+    "IETF LSR - Link State Routing Working Group";
+  contact
+    "
+    WG Web:
+      <https://datatracker.ietf.org/group/lsr/about/>
+    WG List:
+      <mailto: lsr@ietf.org>
+
+    Editor: Srinath Dontula
+        <mailto: sd947e@att.com>
+    Author: Tony Li
+        <mailto: tony.li@tony.li>
+    Author: Peter Psenak
+        <mialto: ppsenak@cisco.com>
+    Author: Les Ginsberg
+        <mailto: ginsberg@cisco.com>
+    Author: Huaimo Chen
+        <mailto: hcen@futurewei.com>
+    Author: Tony Przygienda
+        <mailto: prz@juniper.net>
+    Author: Dave Cooper
+        <mailto: dave.cooper@centurylink.com>
+    Author: Luay Jalil
+        <mailto: luay.jalil@verizon.com>
+    Author: Athar Siddiqui
+        <mailto: athar@att.com>
```

Comment:
IMHO, this can get trimmed to be Srinath, you, and myself.  While the others did author the base document, they haven't contributed here.

Reply:
Will do.

Reply:
I'll do this